Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/OverlayMessagingService.java

```diff
@@ -157,73 +180,191 @@ private void setDataToViews(String bundle) {
             buttonOk.setVisibility(buttonOkVisibility ? View.VISIBLE : View.GONE);
             buttonCancel.setVisibility(buttonCancelVisibility ? View.VISIBLE : View.GONE);
             imageView.setVisibility(imageVisibility ? View.VISIBLE : View.GONE);
+            try {
+                setDataToMediaView(data);
+            } catch (Exception e){
+                RideRequestUtils.firebaseLogEventWithParams("exception", "CONSTRUCT_MEDIA_VIEW", Objects.requireNonNull(e.getMessage()).substring(0, 40), this);
+            }
+            Boolean showContactSupport = supportPhoneNumber != null;
+            if (showContactSupport) {
+                buttonCancelRight.setVisibility(View.VISIBLE);
+                cancelButtonImage.setVisibility(View.VISIBLE);
+                secondaryActions = secondaryActions != null ? secondaryActions.put("CALL_SUPPORT") : new JSONArray("CALL_SUPPORT");
```

Comment:
If it will have only `String` is it possible to use normal array instead of `JSONArray`